test/ParseableInterface/inherited-defaults-execution.swift

```diff
@@ -0,0 +1,47 @@
+// REQUIRES: executable_test
+// RUN: %empty-directory(%t)
+
+// 1) Build the 'Inherited' library and its interface from this file
+//
+// RUN: %target-build-swift-dylib(%t/%target-library-name(Inherited)) -emit-module-path %t/Inherited.swiftmodule -emit-parseable-module-interface-path %t/Inherited.swiftinterface -module-name Inherited %s
+// RUN: rm %t/Inherited.swiftmodule
+
+// 2) Check the interface includes the synthesized initializers of the base
+//    class in the derived class explicitly and uses the '= super' syntax to
+//    inherit its default arguments.
+//
+// RUN: cat %t/Inherited.swiftinterface | %FileCheck --check-prefix=INTERFACE %s
+//
+// INTERFACE: public class Base {
+// INTERFACE:   public init(x: Swift.Int = 45, y: Swift.Int = 98)
+// INTERFACE: }
+// INTERFACE: public class Derived : Inherited.Base {
+// INTERFACE:   override public init(x: Swift.Int = super, y: Swift.Int = super)
+// INTERFACE: }
+
+// 4) Generate a main.swift file that uses the 'Inherited' library and makes use
+//    of the inherited default arguments
+//
+// RUN: echo "import Inherited" > %t/main.swift
+// RUN: echo "print(Derived().x)" >> %t/main.swift
+// RUN: echo "print(Derived().y)" >> %t/main.swift
+
+// 5) Build and run the executable, checking the defaulted arguments resulted in
+//    the correct values being stored
+//
+// RUN: %target-build-swift -I%t -L%t -lInherited -o %t/main %target-rpath(%t) %t/main.swift -swift-version 5
+// RUN: %target-codesign %t/main %t/%target-library-name(Inherited)
+// RUN: %target-run %t/main | %FileCheck --check-prefix=OUTPUT %s
```

Comment:
Oops, look like this broke device testing. In order to make sure libInherited.dylib gets copied to the device, you have to pass it as an argument to `%t/main`. (That's the hack we came up with, anyway: pretend it's an input.)

Reply:
Oh, you're way ahead of me. :-)

Reply:
Yeah, I wasn't sure what I was missing, but @harlanhaskins helped me out!